engine/standalone/util/src/main/scala/pl/touk/nussknacker/engine/standalone/utils/customtransformers/StandaloneCustomExtractor.scala

```diff
@@ -0,0 +1,40 @@
+package pl.touk.nussknacker.engine.standalone.utils.customtransformers
+
+import cats.data.Validated.Valid
+import pl.touk.nussknacker.engine.api._
+import pl.touk.nussknacker.engine.api.context.ContextTransformation
+import pl.touk.nussknacker.engine.api.context.ProcessCompilationError.NodeId
+import pl.touk.nussknacker.engine.standalone.api.StandaloneCustomTransformer
+import pl.touk.nussknacker.engine.standalone.api.types.InterpreterType
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object StandaloneCustomExtractor extends CustomStreamTransformer {
+
+  @MethodToInvoke(returnType = classOf[Void])
+  def invoke(@ParamName("expression") expression: LazyParameter[Any],
+             @OutputVariableName outputVariableName: String)
+            (implicit nodeId: NodeId): ContextTransformation = {
+    ContextTransformation
+      .definedBy(ctx => ctx.withVariable(outputVariableName, expression.returnType))
+      .implementedBy(
+        new StandaloneCustomExtractor(outputVariableName, expression))
+  }
+
+}
+
+class StandaloneCustomExtractor(outputVariableName: String, expression: LazyParameter[Any]) extends StandaloneCustomTransformer {
+
+  override def createTransformation(outputVariable: String): StandaloneCustomTransformation =
```

Comment:
guess we can now change signature of StandaoneCustomTransformer - we don't need outputVariable now?

Reply:
I want to leave this for backward compatibility or second option to create another `StandaloneCustomTransformer` next to this. But I'm not sure if I correctly understand this standalone api. E.g. I was confused that for now you can't create CustomTransformer that just filtering...